apps/drupal-default/particle_theme/templates/layout/header/custom-header.html.twig

```diff
@@ -4,6 +4,24 @@
  * Custom Header.
  */
 #}
+
+{%
+  set classes = [
+    'flex',
+    'justify-center',
+    'py-2',
+    'hide-under-1200',
```

Comment:
needed a custom class here for a media query that was on the original classes, to hide the element under 1200px. I don't know how to do this in Tailwind yet.